docs/source/quickstart.rst

```diff
@@ -124,3 +124,29 @@ Here, we'll process our data in the background, while responding immediately to
         resp.media = {'success': True}
 
 A ``POST`` request to ``/incoming`` will result in an immediate response of ``{'success': true}``.
+
+
+Here's a sample code to post a file with background::
+
+    @api.route("/")
+    async dev upload_file(req, resp):
```

Comment:
I believe you meant `def` instead of `dev`

Reply:
Thanks! That was my mistake!